<?xml version="1.0" encoding="UTF-8"?><testrun name="TestWordCount (1)" project="mrunit" tests="3" started="3" failures="1" errors="0" ignored="0">
  <testsuite name="stubs.TestWordCount" time="2.302">
    <testcase name="testReducer" classname="stubs.TestWordCount" time="1.792"/>
    <testcase name="testMapper" classname="stubs.TestWordCount" time="0.116"/>
    <testcase name="testMapReduce" classname="stubs.TestWordCount" time="0.394">
      <failure>java.lang.AssertionError: 3 Error(s): (Matched expected output (dog, 1) but at incorrect position 2 (expected position 1), Matched expected output (cat, 2) but at incorrect position 1 (expected position 0), Received unexpected output (1, 1) at position 0.)
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.hadoop.mrunit.TestDriver.validate(TestDriver.java:294)
	at org.apache.hadoop.mrunit.MapReduceDriverBase.runTest(MapReduceDriverBase.java:142)
	at org.apache.hadoop.mrunit.TestDriver.runTest(TestDriver.java:137)
	at stubs.TestWordCount.testMapReduce(TestWordCount.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</failure>
    </testcase>
  </testsuite>
</testrun>
